openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -35,9 +35,34 @@ <h3 class="section-title">${ section['display_name'] }</h3>
                                 >
                                     <div class="subsection-text">
                                         ## Subsection title
-                                        <span class="subsection-title">${ subsection['display_name'] }</span>
-
+                                        <span class="subsection-title">
+                                            % if subsection['id'] in milestones:
```

Comment:
What's the latest perspective on `milestones` versus `gated content` terminology?  Since GatedContent is the new terminology used for XBlock services and governs Prereq course content, why not continue to use that terminology in all the higher layer code that supports Gated content?  This way, milestones can remain as an encapsulated implementation detail within Gated Content.

Reply:
@nasthagiri 
Good point. I changed this to gated_content.